code/frameworks/sveltekit/src/mocks/app/stores.ts

```diff
@@ -0,0 +1,34 @@
+import { getContext, setContext } from 'svelte';
+
+function createMockedStore(contextName: string) {
+  return [
+    {
+      subscribe(runner: any) {
+        const page = getContext(contextName);
+        runner(page);
+        return () => {};
+      },
+    },
+    (value: unknown) => {
+      setContext(contextName, value);
+    },
+  ] as const;
+}
+
+export const [page, setPage] = createMockedStore('page-ctx');
+export const [navigating, setNavigating] = createMockedStore('navigating-ctx');
+const [updated, setUpdated] = createMockedStore('updated-ctx');
+
+Object.defineProperty(updated, 'check', {
+  value: () => {},
+});
```

Comment:
Yeah it was solely to please typescript...do you prefer this other syntax?

Reply:
I think `(updated as any).check = () => {};` (or similar) makes this clearer. `Object.define..` would be something that in a year I would be afraid to touch because I didn't fully know why we did it this way, where my suggestion would show that it's just TS that's complaining.